tests/test_06.yml

```diff
@@ -0,0 +1,6 @@
+global_size: { height: 60, width: 12 }
+
+pdi:
+  plugins:
```

Comment:
I suggest you put hdf5 plugins here, to follow the case where PDI was enabled and did output. Also we can be sure from the test that the plugin is not triggered in the no-pdi case.